services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -62,20 +63,22 @@
     final Set<Integer> _electedLeaders;
     final Collection<Broker> _allBrokers;
     final double _partitionToBrokerRatioThreshold;
+    final int _topicReplicationFactor;
```

Comment:
Done.

The name change was an attempt to show the difference between the topics current replication factor and the replication factor passed in as a config.